src/services/login-util.js

```diff
@@ -17,6 +18,7 @@ const generateAuthenticationUrl = (codeChallenge, scope, state) => {
     redirect_uri: 'http://localhost:4242',
     scope: scope,
     state: state,
+    audience: ['api-dev.checkly.run'],
```

Comment:
This is different for dev/test and prod

Reply:
You are right, will move this to config file.